turbopack/crates/turbo-tasks/src/vc/mod.rs

```diff
@@ -446,8 +442,7 @@ where
     /// Returns `None` if the underlying value type is not a `K`.
     pub async fn try_resolve_downcast<K>(vc: Self) -> Result<Option<Vc<K>>, ResolveTypeError>
     where
-        K: Upcast<T>,
-        K: VcValueTrait + ?Sized + Send,
```

Comment:
Repeating the same left-hand-side for a where clause is valid, but unconventional. Converting this to use `+` instead to make it easier to read.